pkgs/by-name/co/cosmic-applets/package.nix

```
@@ -88,11 +88,7 @@ rustPlatform.buildRustPackage (finalAttrs: {
     homepage = "https://github.com/pop-os/cosmic-applets";
     description = "Applets for the COSMIC Desktop Environment";
     license = lib.licenses.gpl3Only;
-    maintainers = with lib.maintainers; [
-      qyliss
-      nyabinary
-      HeitorAugustoLN
-    ];
+    maintainers = lib.teams.cosmic;
```

Actually 

```suggestion
    maintainers = lib.teams.cosmic.members;
```

You can verify with a Nix repl

---

Oops, I've missed that! Thanks